PowerEditor/installer/nativeLang/german.xml

```diff
@@ -963,11 +964,11 @@ Translation note:
 					<Item id="6106" name="Dateireiter (Tabs)"/>
 					<Item id="6107" name="Verkleinern"/>
 					<Item id="6108" name="Tabs nicht mit der Maus verschiebbar"/>
-					<Item id="6109" name="Inaktive Tabs dunkler darstellen"/>
+					<Item id="6109" name="Farbe für Inaktive Tabs ändern"/>
```

Comment:
"Inaktive Tabs" should be lower case: "inaktive Tabs" as it's not at the beginning of a sentence. Or a slightly different wording: "Farbe inaktiver Tabs ändern"